exec-agent/process/common.go

```diff
@@ -0,0 +1,68 @@
+package process
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	DefaultLogsPerPageLimit = 50
+)
+
+func maskFromTypes(types string) uint64 {
+	var mask uint64
+	for _, t := range strings.Split(types, ",") {
+		switch strings.ToLower(strings.TrimSpace(t)) {
+		case "stderr":
+			mask |= StderrBit
```

Comment:
This file is rather webscoket & rest apis commons, i will rename it to _service_commons.go_
in accordance to  _rest_service_.go, _ws_service.go_ names